src/index.tsx

```diff
@@ -6,52 +6,61 @@ import {
 
 import { ICommandPalette, MainAreaWidget } from "@jupyterlab/apputils";
 
-import {IDocumentManager} from "@jupyterlab/docmanager";
+import { IDocumentManager } from "@jupyterlab/docmanager";
 
 import { Widget } from "@lumino/widgets";
 import React from "react";
-import ReactDOM from 'react-dom';
+import ReactDOM from "react-dom";
 
-import SearchPage from './SearchPage';
+import SearchPage from "./SearchPage";
 
 declare namespace JSX {
-  interface Element { }
-  interface IntrinsicElements { div: any}
+  interface Element {}
+  interface IntrinsicElements {
+    div: any;
+  }
 }
 
-function initSearchWidget(content: Widget, docManager: IDocumentManager) {
-  let searchHeader = <div
-    style={{
-      height: '100%'
-    }}
-  >
-    <SearchPage
-      docManager={docManager}
-    />
-  </div>;
+function initSearchWidget(content: Widget, docManager: IDocumentManager, app: JupyterFrontEnd) {
+  let searchHeader = (
+    <div
+      style={{
+        height: "100%"
+      }}
+    >
+      <SearchPage docManager={docManager} app={app}/>
+    </div>
+  );
   ReactDOM.render(searchHeader, content.node);
 }
 
 /**
- * Initialization data for the deepsearch extension.
+ * Initialization data for the jupyter_mandalab extension.
  */
 const extension: JupyterFrontEndPlugin<void> = {
-  id: "deepsearch",
+  id: "jupyter_mandalab",
   autoStart: true,
   requires: [ICommandPalette, IDocumentManager, ILabShell],
-  activate: (app: JupyterFrontEnd, palette: ICommandPalette, docManager: IDocumentManager, labShell: ILabShell) => {
-    console.log("JupyterLab extension deepsearch is activated!");
+  activate: (
+    app: JupyterFrontEnd,
+    palette: ICommandPalette,
+    docManager: IDocumentManager,
+    labShell: ILabShell,
+    
+  ) => {
+    console.log("JupyterLab extension jupyter_mandalab is activated!");
```

Comment:
Just to make sure - You are intentionally log this to the console right?
Or is it something which needs to be deleted and missed?

Reply:
This is intentionally logged (It is a part of Jupyter's cookiecutter template)

Reply:
Great :)